web/js/livemap.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+let geocoder;
+var mapcanvas = $("#map-preview");
+var mapOptions = {
+zoom: 15,
+center: {lat: 50.948352, lng: 3.131108},  //this default to a map of the centre of Roeselare
+streetViewControl: false,
+mapTypeControl: false
+};
+var map = new google.maps.Map(mapcanvas[0], mapOptions);
+var marker;
+
+var codeAddress = function() {
+
+    var address = getAddress();
+    address = address.street + " " +
```

Comment:
Ik heb dit aangepast, maar zit nu nog met het probleem dat de map niet mooi verschijnt in het kadertje. Kan iemand mij hierbij helpen, aangezien ik niet weet waar de CSS voor dit element zich bevindt.